src/verilogAST.cpp

```diff
@@ -259,7 +258,7 @@ std::string Always::toString() {
   for (auto statement : body) {
     always_str +=
         variant_to_string<BehavioralStatement *, Declaration *>(statement) +
-        ";\n";
+        "\n";
```

Comment:
Again, the logic for generating the body of an always block is inconsistent with the logic for a module definition, we should use the same pattern for both (either everyone generates their own semicolon, or the enclosing construct adds the semicolons)